scripts/helpers/slashes.js

```diff
@@ -1,5 +1,5 @@
 'use strict'
 
 module.exports = function (str) {
-  return process.platform === 'win32' && str.replace(/\\/g, '/')
+  return str.replace(/\\/g, '/')
```

Comment:
Alternatively, if you still want the check I can changed it to a ternary.

That being said, this way is consistent with one more case we have to replace the slashes where we don't check for process.